TabGroup/TabGroup.js

```diff
@@ -35,21 +35,24 @@ const TabBase = kind({
 	},
 
 	computed: {
-		tabLabel: ({children, className, css, icon, labelPosition, orientation, selected, ...rest}) => {
-			let inline;
+		tabLabel: ({children, className, css, icon, labelPosition, orientation, index, ...rest}) => {
```

Comment:
I think I need a walkthrough of the intent of the changes to TabGroup before reviewing.

Reply:
Because my assumption was that `selected` is a boolean prop, I ran into a bit of confusion when `selected` was being used as if it's an `index` prop.
I went ahead with these changes to keep it consistent where `index` is the current `index` selected, and reserve `selected` as a boolean prop, which we do not need at the moment or AFAIK.

Reply:
I see. maintaining consistency between components for prop names (Button, ToggleButton, etc use `selected` as a boolean). How would you feel about updating TabGroup to use `selectedIndex` rather than `index` so it's a little more descriptive?

I think that `Group` and `Repeater` have some props that let consumers more easily customize the prop name which may help reduce the number of changed lines here. Worth looking into. If the theme doesn't directly rely on these changes, we could put them in a separate PR to help compartmentalize the review. WDYT?

Reply:
Sounds good `selectedIndex` is better.